docs/source/starters/new_project_tools.md

```diff
@@ -1,4 +1,229 @@
-# Configure a new project
+# Tools to customise a new Kedro project
 
-<!--TO DO-->
-<!--Detailed usage of add-ons goes here-->
+There are several ways to configure your new project with the tools and example code.
+
+## `kedro new`
+
+Navigate to the directory in which you would like to create your new Kedro project, and run the following command:
+
+```bash
+kedro new
+```
+
+This will start the new project creation workflow.
+
+``` {note}
+You can also add flags to `kedro new` to skip some or all of the steps in the project creation workflow to skip queries about how you want to customise the project. The flags are described below.
+```
+
+### Project name
+The first prompt asks you to input a project name.
+
+You can skip the step to name the project by adding it to your command. For example:
+
+```bash
+kedro new --name=spaceflights
+```
+
+### Tools
+You are then asked to select which tools to include. Choose from the list using comma separated values `(1,2,4)`, a range of values `(3-5)`, or the key words `all` or `none`. Skipping the prompt by entering no value will result in the default selection of `none`. Further information about each of the tools is described below in [Kedro tools](#kedro-tools).
+
+
+| ![](../meta/images/kedro-new-tools-prompt.png)|
+|:--:|
+| *Prompt for tools selection* |
+
+
+You may also specify your tools selection directly from the command line by using the flag `--tools`:
+
+```bash
+kedro new --tools=<your tool selection>
+```
+
+To specify your desired tools you must provide them by name as a comma separated list, for example `--tools=lint,test,viz`. The following tools are available for selection: `lint`, `test`, `log`, `docs`, `data`, `pyspark`, and `viz`.
+
+A list of available tools can also be accessed by running `kedro new --help`
+
+![](../meta/images/kedro-new-help.png)
+
+
+### Example code
+In the final step you are asked whether you want to populate the project with an example pipeline. Further information about this pipeline is below in [Example pipeline](#example-pipeline).
+
+You can add the example pipeline to your new project as follows:
+
+```bash
+kedro new --example=y
+```
+
+## `kedro new --config=`
+
+You can also supply values to `kedro new` by providing a YML configuration file to your `kedro new` command. Consider the following file:
+
+```yaml
+# config.yml
+
+"project_name":
+    "My Project"
```

Comment:
I was reading the document and it was not immediately clear to me that now it was describing a completely separate option, and when I got here I immediately thought "how does this interact with `kedro new --name=`?".

Maybe to keep the flow of the text, it would be necessary to add something like "apart from the individual tools options, you can supply all the customisation options in one `config.yml`" or something like that?

Reply:
Addressed in [169c6be](https://github.com/kedro-org/kedro/pull/3331/commits/169c6be359ace33b3e7f46ad6b7502f0cba90d74)